bundles/binding/org.openhab.binding.dsmr/src/main/java/org/openhab/binding/dsmr/internal/DSMRPort.java

```diff
@@ -276,13 +285,77 @@ private void handlePortState() {
+                portSettings = DSMRPortSettings.LOW_SPEED_SETTINGS;
+            } else {
+                portSettings = DSMRPortSettings.HIGH_SPEED_SETTINGS;
+            }
+            logger.debug("Switched port settings to:" + portSettings);
+        } else {
+            portSettings = fixedPortSettings;
+            logger.info("Fixed port settings configured (autodetect DISABLED):" + portSettings);
+        }
+    }
+
+    /**
+     * Checks if the given port name is autodetected by gnu.io or already listed
+     * in the system property gnu.io.rxtx.SerialPorts
+     *
+     * @param portName String containing the port name to lookup
+     * @return true if port exists, false otherwise
+     */
+    private boolean portExists(String portName) {
+        @SuppressWarnings("unchecked")
+        Enumeration<CommPortIdentifier> portEnum = CommPortIdentifier.getPortIdentifiers();
+
+        boolean portExists = false;
+
+        logger.debug("Searching autodetected ports for:" + portName);
+        while (portEnum.hasMoreElements()) {
+            CommPortIdentifier portIdentifier = portEnum.nextElement();
+            if (portIdentifier.getPortType() == CommPortIdentifier.PORT_SERIAL) {
+                logger.debug("Found serial port:" + portIdentifier.getName());
+                if (portIdentifier.getName().equals(portName)) {
+                    portExists = true;
+                }
+            }
+        }
+        if (!portExists) {
+            Properties properties = System.getProperties();
+            String currentPorts = properties.getProperty("gnu.io.rxtx.SerialPorts", "");
+            if (currentPorts.indexOf(portName) >= 0) {
+                logger.debug(portName + " is listed in system property gnu.io.rxtx.SerialPorts");
+
+                portExists = true;
+            } else {
+                logger.debug(portName + " is not listed in system property gnu.io.rxtx.SerialPorts");
+            }
+        }
+        return portExists;
+    }
+
+    /**
+     * Adds port name to the system property gnu.io.rxtx.SerialPorts if it is not
+     * already present
+     *
+     * @param portName String containing the port name to add
+     */
+    private void addPortToSystemProperty(String portName) {
+        Properties properties = System.getProperties();
+        String currentPorts = properties.getProperty("gnu.io.rxtx.SerialPorts", "");
+        if (currentPorts.indexOf(portName) >= 0) {
+            logger.warn(portName + " is already listed in system property gnu.io.rxtx.SerialPorts");
+        } else {
+            logger.info("Adding port " + portName + " to system property gnu.io.rxtx.SerialPorts");
+            if (currentPorts.length() == 0) {
+                properties.setProperty("gnu.io.rxtx.SerialPorts", portName);
+            } else {
+                properties.setProperty("gnu.io.rxtx.SerialPorts", currentPorts + File.pathSeparator + portName);
+            }
```

Comment:
Isn't the preferred way to do this documented [here](https://github.com/openhab/openhab/wiki/symlinks)?